app/javascript/components/Modal.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from "react";
```

Comment:
Consider putting this and style in their own folder 📁 

`app/javascript/components/modal/Modal.tsx`
`app/javascript/components/modal/ModalStyle.ts`